### ISSUE TITLE
Picker TitleColor not working on iOS

### ISSUE THREAD

**Issue by doriangh** [dotnet/maui#19191]
### Description

When setting the TitleColor in the Picker control, the color is always Black

### Steps to Reproduce

1. Create a new MAUI app
2. Add a Picker control on the MainPage, setting the Title and TitleColor
3. Run and observe

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Did you find any workaround?

Not a workaround, but having no items in the source of the picker, opening it, then closing it does apply the TitleColor. Selecting an item, then clearing it does not have the same outcome (applying the TitleColor as does closing the picker without items).

### Relevant log output

```shell
This still does not work in the latest Nightly build
```

**Comment by MichaelShapiro** [comment]
I'm experience identical problem with the Picker's TitleColor on iOS.

**Comment by XamlTest** [comment]
Verified this on Visual Studio Enterprise 17.9.0 Preview 2(8.0.3). Repro on iOS 17.0 and MacCatalyst, not repro on Windows 11 and Android 14.0-API34 with below Project:
[19191.zip](https://github.com/dotnet/maui/files/13857878/19191.zip)

**Comment by kubaflo** [comment]
@jsuarezruiz @doriangh here's a possible fix: https://github.com/dotnet/maui/pull/20205

**Comment by marcoablanco** [comment]
Still happend in iOS in net9

**Comment by kubaflo** [comment]
@marcoablanco Yeah I'd be surprised if it started working without this pr lol https://github.com/dotnet/maui/pull/20205
